FIAT/hct.py

```diff
@@ -1,15 +1,20 @@
 from FIAT.functional import (PointEvaluation, PointDerivative,
+                             IntegralMoment,
                              IntegralMomentOfNormalDerivative)
 from FIAT import finite_element, dual_set, macro, polynomial_set
 from FIAT.reference_element import TRIANGLE, ufc_simplex
+from FIAT.quadrature import FacetQuadratureRule
 from FIAT.quadrature_schemes import create_quadrature
-from FIAT.jacobi import eval_jacobi
+from FIAT.jacobi import eval_jacobi, eval_jacobi_batch, eval_jacobi_deriv_batch
 
 
 class HCTDualSet(dual_set.DualSet):
```

Comment:
Maybe a citation for which paper we're pulling from?  Also, these are defined on incenter splits in the paper but we've implemented barycenter splits.  It's not clear what the practical limitations are, but that should be examined and documented somewhere (here or FInAT).

Reply:
I added the reference in the docstring, and I also added the proper citation in FInAT.